src/useValidation/useValidation.ts

```diff
@@ -52,13 +56,22 @@ export function useValidation<T>(
     return valid.value
   }
 
-  const validation: UseValidation = {
+  const state = reactive({
```

Comment:
One last thought. Would it make more sense for this reactive object to be named `validation`? `state` matches up nicely with prefect-design but it make less sense here as a standalone thing. I don't feel strongly, approving this either way. 

```typescript
const { validation } = useValidation()
```

Reply:
I think I like `state` more, even without considering it's alignment with how we currently use vee-validate in prefect-design.